ONSdigital/ssdc-rm-dockerfiles
# 1083 Build multiple python pipenv versions

# Motivation and Context

We want multiple python versions of our python-pipenv image

# What has changed



- Make the python base image dynamic, include the two digests we want to build 3.10 and 3.11 from

# How to test?



Check the build still works, and the two included tags are correct and up to date.

# Links



https://trello.com/c/na17GJF3/1083-update-our-python-pipenv-image-and-builds-to-build-multiple-versions
